specification/cognitiveservices/HealthInsights/healthinsights.autorest/cadl-project.yaml

```diff
@@ -0,0 +1,10 @@
+emit: [
```

Comment:
@catalinaperalta should we trying to start using typespec instead of cadl?

Reply:
Also Any reason we have a healthinsights.autorest folder? What is the purposes of this folder? For general repository structure please see https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md.

Reply:
I'll try to take a look at whether moving on to the typespec packages would affect anything or if it can be done quickly...if it is simple I'll open a pr that @asaflevi-ms can merge into this branch.

Reply:
>About the healthinsights.autorest folder the service team said that they need a full swagger definition of their service, so they generate their swagger from this folder. I do think it should be renamed to HealthInsights.OpenApi to be clearer.

I would like to understand that a little more as generally I would expect the openapi output to be included in the same projects as the SDK's. Why do we need this extra wrapper?